.github/workflows/Repology.yml

```diff
@@ -30,22 +30,20 @@ jobs:
       - name: Create Pull Request
         id: cpr
         uses: peter-evans/create-pull-request@v7
-        env:
-          GH_TOKEN: ${{ secrets.CREW_PR_TOKEN }}
         with:
+          token: ${{ secrets.CREW_PR_TOKEN }}
           add-paths: "tools/repology.json"
           title: "Update Repology JSON"
           body: "Automatic PR to update tools/repology.json"
           author: "github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>"
           commit-message: "Update Repology JSON"
           delete-branch: true
-          team-reviewers: chromebrew/active
```

Comment:
Why is this removed?

Reply:
The gh command is adding the team to the reviewer list below...

Reply:
Gotcha.